react/src/lib/components/SubsurfaceViewer/layers/labeled_points/labeledPointsLayer.ts

```diff
@@ -0,0 +1,146 @@
+import { CompositeLayer, UpdateParameters } from "@deck.gl/core/typed";
```

Comment:
`labeledPointLayer` seems a bit strange to me. Why not simply `pointLayer` ?

Reply:
This is some groundwork for the future. This layer will also be used to display text labels with/instead of cicle points.

Reply:
But you do not add the `circle` feature in the name ;)
Like the grid2d layer does not have the `contour` feature in its name

I really think that the current name is more misleading than helpful

Reply:
Renamed.